js/projects.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var projects = [];
+
+function Project(name, startDate, endDate, description, link){
+  this.name = name,
+  this.startDate = startDate,
+  this.endDate = endDate,
+  this.description = description,
+  this.link = link,
+  projects.push(this)
```

Comment:
this works, but its typically not the best idea to use methods inside of your constructor. its better practice to place it outside of the constructor on the prototype.